src/modules/dashboard/components/Dialogs/CreatePaymentDialog/CreatePaymentDialog.tsx

```diff
@@ -161,15 +202,19 @@ const CreatePaymentDialog = ({
         if (formValues.values.forceAction !== isForce) {
           setIsForce(formValues.values.forceAction);
         }
+
         return (
           <Dialog cancel={cancel}>
             <DialogForm
               {...formValues}
               colony={colony}
               isVotingExtensionEnabled={isVotingExtensionEnabled}
               back={() => callStep(prevStep)}
-              subscribedUsers={colonyMembers?.subscribedUsers || []}
+              subscribedUsers={filteredVerifiedRecipients}
               ethDomainId={ethDomainId}
+              showWhitelistWarning={getIsUnverifiedRecipient(
+                formValues.values?.recipient?.profile?.walletAddress,
```

Comment:
I'm not totally sure about this, as I have a suspicion the `formValue` you get here is actually the previous state _(value)_, so this might screw with the warning display logic.

Please double check it _(with multiple values)_ to make sure it works properly

Reply:
Just making sure. I trust when you say it's working as intended :+1: